cabal.haskell-ci

```diff
@@ -61,6 +61,13 @@ constraint-set deepseq-1.4
   ghc: (>= 7.8 && <7.10) || == 8.2.2
   constraints: deepseq ==1.4.*
 
+-- Copy over additional properties specified in a cabal.project file.
+-- Possible values are:
+--   none: Do not copy any properties from the cabal.project file.
+--   some: Copy particular properties (???) from the cabal.project file. (Default)
```

Comment:
It's rather unclear to me which properties `some` copies over. Feedback on the documentation for this would be appreciated.